cmd/gitserver/server/server.go

```diff
@@ -1104,7 +1106,7 @@ func (s *Server) handleSearch(w http.ResponseWriter, r *http.Request) {
 
 	var latencyOnce sync.Once
 	matchesBuf := streamhttp.NewJSONArrayBuf(8*1024, func(data []byte) error {
-		tr.LogFields(otlog.Int("flushing", len(data)))
+		tr.AddEvent("LogFields", attribute.Int("flushing", len(data)))
```

Comment:
Let's take this opportunity to give these useful values as well! The first field is an event name, so this could look like:

```suggestion
		tr.AddEvent("flushing data", attribute.Int("data.len", len(data)))
```